socorro/signature/rules.py

```diff
@@ -627,15 +651,14 @@ class SigFixWhitespace(Rule):
     * reduce consecutive spaces to a single space
 
     """
-
     WHITESPACE_RE = re.compile(r'\s')
     CONSECUTIVE_WHITESPACE_RE = re.compile(r'\s\s+')
 
     def predicate(self, crash_data, result):
-        return isinstance(result.get('signature'), six.string_types)
+        return isinstance(result.signature, six.string_types)
```

Comment:
This seems silly since the signature is always a str and doesn't change. We should probably nix this.